.evergreen/serverless/create-instance.sh

```diff
@@ -46,6 +46,17 @@ if [ -z "$SERVERLESS_INSTANCE_NAME" ]; then
     SERVERLESS_INSTANCE_NAME="$RANDOM-DRIVERTEST"
 fi
 
+# Ensure that a Python binary is available for JSON decoding
+# shellcheck source=.evergreen/find-python3.sh
+. ../find-python3.sh || return
```

Comment:
This is related to my question in https://github.com/mongodb-labs/drivers-evergreen-tools/pull/255#discussion_r1052086238. I'm not sure why some other scripts (e.g. `start-orchestration.sh`) used `$DRIVER_TOOLS` instead of a relative path, but I think this is correct in this context.

Also, do I assume correctly that it's still preferable to use `find-python3.sh` even if we don't need to create a venv? This script only utilizes built-in Python modules (`sys` and `json`) so I don't think there's any need for a venv. I realize `find-python3.sh` requires the discovered Python environment to support venvs, but I assume that's not really a concern on Evergreen.

Reply:
Noted. Per https://github.com/mongodb-labs/drivers-evergreen-tools/pull/256#discussion_r1052853294 I left the script's use of `$0` as-is since it is never sourced.